packages/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.3.4+1
+
+* Fix the bug that `SKPaymentQueueWrapper.transactions` doesn't return all transactions.
+* Fix the app crashes  if `queryPastPurchases` is called in the `main()`.
```

Comment:
nits:
maybe add iOS?
Also, i don't think it crashes if `queryPastPurchases` is called in the `main()`.
It crashes if the InAppPurchaseConnection instance is created on main.

Reply:
Updated

Reply:
lgtm! THANKS